engines/python/setup/djl_python/rolling_batch/vllm_rolling_batch.py

```diff
@@ -27,21 +27,19 @@ def __init__(self, model_id_or_path, device, properties, **kwargs):
         :param properties: other properties of the model, such as decoder strategy
         """
         super().__init__(-1, **kwargs)
-        self.dtype = kwargs.pop("dtype", 'auto')
+        self.dtype = properties.get("dtype", 'auto')
```

Comment:
should we also change handlers to pass these parameters in `properties`?

Reply:
they already in properties